materials/tasks.py

```diff
@@ -0,0 +1,19 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from config.settings import EMAIL_HOST_USER
+from users.models import Subscription, User
+
+
+@shared_task
+def sendmail_course_updated(course):
```

Comment:
Во вьюшке в таску ты передаешь id, а значит тут тоже нужно принимать id
```suggestion
def sendmail_course_updated(course_id):
```